src/components/Article.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Article = ({ title, date, text }) => (
+  <>
+    <h1>{title}</h1>
+    <span>{date}</span>
+    <p>{text}</p>
+  </>
+);
+
+Article.defaultProps = {
+  date: new Date(),
```

Comment:
```suggestion
  date: new Date().toString(),
```
Если не передать параметр date будет ошибка, потому что new Date() вернет обьект, а его нельзя вставлять в jsx.